### PR TITLE
Feature#8 my flights design

### LINKED ISSUE #8: My Flights Design
# REQUIREMENTS
As a user, I want to see a list of flights made showing the city of origin from which I departed, the destination city I arrived at, the date I made the trip and how many passengers I registered.

In addition, a button should appear at the bottom of the screen that allows me to add new flights.

# DESIGN
imagen

# Acceptance Criteria
List all required acceptance criteria
- [ ] Title
- [ ] FlatList where the saved flights from the database are shown.
- [ ] Fixed button at the bottom where a plus icon is displayed.

### PR DESCRIPTION
**Pull request by Alonso-AS**
# Description
The design of the 'My Flights' screen was made, which will contain the information saved in Firebase, of the flights of each user who has entered through the login.
 
 
  
  ## Screenshots 
image

### CODE REVIEW

**Review by jkzasori — APPROVED** [review]
For me it's okay!

File: FlightReservation/components/screens/ItemsMyFlights.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <View style={MyFlightsStyles.column}>
+          <Text style={MyFlightsStyles.fontBold}>BEG</Text>
+          <Text style={MyFlightsStyles.fontNormal}>Serbia</Text>
+        </View>
+        <View style={[MyFlightsStyles.column, MyFlightsStyles.centerVertical]}>
+          <CrossPlatformIcon
+            name="airplane"
+            size={25}
+            styles={MyFlightsStyles.centerHorizontal}
+            color="rgba(74,144,226,1)"
```
Comment (OrlandoAlPe) [inline FlightReservation/components/screens/ItemsMyFlights.js:19]: I recomend to change the color of the + icon button and the plane icon from  color="rgba(74,144,226,1)" to 
color ="rgba(92,110,248,1)" which is the title color to have a consisting color palette.

File: FlightReservation/components/screens/ItemsMyFlights.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import CrossPlatformIcon from 'react-native-cross-platform-icons';
+import MyFlightsStyles from '../assets/styles/MyFlightsStyles';
+
+const ItemsMyFlights = ({item}) => {
+  return (
+    <>
```
Comment (OrlandoAlPe) [inline FlightReservation/components/screens/ItemsMyFlights.js:8]: This empty brackets doesn't seem to have a functionality.

**Review by OrlandoAlPe — APPROVED** [review]
LGTM, just two small details:

- Color consistency
- Empty bracktes,

File: FlightReservation/components/screens/ItemsMyFlights.js (hunk 2 above)
Comment (jkzasori) [reply FlightReservation/components/screens/ItemsMyFlights.js:8]: That is a fragment in react and you can use for not add extra multiple nodes to the DOM I think they are using that for not use an extra view... 
https://es.reactjs.org/docs/fragments.html

File: FlightReservation/components/screens/ItemsMyFlights.js (hunk 1 above)
Comment (jkzasori) [reply FlightReservation/components/screens/ItemsMyFlights.js:19]: I agree with you

**Pull request merged**